iphone/cli/commands/_build.js

```diff
@@ -7142,6 +7142,23 @@ function sha1(value) {
 	return crypto.createHash('sha1').update(value).digest('hex');
 }
 
+// This function has the advantage to detect bridges architectures
+// like x86_64 in Rosetta mode (process.arch would still print "arm64" in that case)
+async function processArchitecture() {
+	return new Promise((resolve, reject) => {
+		// eslint-disable-next-line security/detect-child-process
+		const exec = require('child_process').exec;
+
+		exec('uname -m', function (error, stdout) {
+			if (error) {
+				reject(error);
+				return;
+			}
+			resolve(stdout);
```

Comment:
I think `stdout` needs to be trimmed here? On my machine `child_process.exec('uname -m', (err, stdout) => console.log(stdout === 'x86_64'))` returns false but `child_process.exec('uname -m', (err, stdout) => console.log(stdout.trim() === 'x86_64'))` returns true and I assume the same is true for arm64 machines

Reply:
Interesting. Good point!